tests/controls/python_spec.rb

```diff
@@ -0,0 +1,39 @@
+title 'Python & Libraries'
+
+control 'Python version' do
+  impact 'high'
+  title 'Python should be a specific version'
+  desc 'Python should be a specific version'
+  tag 'python'
+  tag 'version'
+
+  describe command('python --version') do
+    its('stdout') { should match /Python 3.7.1/ }
+    its('exit_status') { should eq 0 }
+  end
+end
+
+control 'Pandas can read a CSV' do
+  impact 'high'
+  title 'Python Pandas is installed and can read a CSV'
+  desc 'Python Pandas is installed and can read a CSV'
+  tag 'python'
+
+  describe command('python /tests/pandas_read_csv.py') do
+    its('stdout') { should match /foo bar baz/ }
+    its('exit_status') { should eq 0 }
+  end
+end
+
+control 'Pandas can read a CSV from S3' do
+  impact 'high'
+  title 'Python Pandas is installed and can read a CSV from s3'
+  desc 'Python Pandas is installed and can read a CSV from s3'
```

Comment:
Most RStudio users wouldn't use pandas to read a CSV, but I guess we're testing getting AWS creds from kube2iam?

Reply:
We're not right now. I was just looking at a generic scenario that @RobinL gave me a few weeks ago. I need to track down the RStudio users to get another scenario or two 👍